app/views/shared/check_answers/_dependants.html.erb

```diff
@@ -1,66 +1,16 @@
-      </div>
-    </div>
-
-    <dl class="govuk-summary-list">
-
-      <%= check_answer_link(
-              name: :dependant,
-              url: nil,
-              question: t('providers.means_summaries.show.dependants.name'),
-              answer: dependant.name,
-              read_only: true,
-              align_right: false
-          ) %>
-      <%= check_answer_link(
-              name: :dependant,
-              url: nil,
-              question: t('providers.means_summaries.show.dependants.date_of_birth'),
-              answer: dependant.date_of_birth,
-              read_only: true,
-              align_right: false
-          ) %>
-      <%= check_answer_link(
-              name: :dependant,
-              url: check_answer_url_for(:providers, :dependants_relationships, @legal_aid_application, dependant_id: dependant),
-              question: t('providers.means_summaries.show.dependants.relationship'),
-              answer: dependant.relationship.nil? ? '' : t("shared.forms.dependants.relationship.option.#{dependant.relationship}"),
-              read_only: false,
-              align_right: false
-          ) %>
-      <%= check_answer_link(
-              name: :dependant,
-              url: check_answer_url_for(:providers, :dependants_full_time_educations, @legal_aid_application, dependant_id: dependant),
-              question: t('providers.means_summaries.show.dependants.in_full_time_education'),
-              answer: yes_no(dependant.in_full_time_education),
-              read_only: false,
-              align_right: false
-          ) %>
-      <%= check_answer_link(
-              name: :dependant,
-              url: check_answer_url_for(:providers, :dependants_monthly_incomes, @legal_aid_application, dependant_id: dependant),
-              question: t('providers.means_summaries.show.dependants.income'),
-              answer: dependant.has_income? ? number_to_currency(dependant.monthly_income) : yes_no(dependant.has_income?),
-              read_only: false,
-              align_right: false
-          ) %>
-      <%= check_answer_link(
-              name: :dependant,
-              url: check_answer_url_for(:providers, :dependants_assets_values, @legal_aid_application, dependant_id: dependant),
-              question: t('providers.means_summaries.show.dependants.assets'),
-              answer: dependant.has_assets_more_than_threshold? ? number_to_currency(dependant.assets_value) : yes_no(dependant.has_assets_more_than_threshold?),
-              read_only: false,
-              align_right: false
-          ) %>
-    </dl>
+  <% @legal_aid_application.dependants.each_with_index do |dependant, index| %>
+    <%= check_answer_one_change_link(
+      name: "dependants_#{index + 1}",
+      url: providers_legal_aid_application_dependant_path(@legal_aid_application, dependant),
+      question: "Dependant #{index + 1}",
+      answer_hash: dependant_hash(dependant),
+      read_only: FALSE
```

Comment:
I like what's done here - very clean. One thing, `FALSE` is a deprecated constant. Prefer `false`

Reply:
🤦 Stupid CAPSLOCK key! :)